### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by Jajaho**
Add blog to production

### CODE REVIEW

File: _posts/2023-12-05-caudri-challenge-2023.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+- **Fifth Place: Team Smart Rollerz (DHBW Stuttgart)**
+
+## Acknowledgments
+
+This event wouldn't have been possible without the dedication of many individuals and organizations:
+
+- The organizing team: Yannik, Marc, Max, Jakob, Niko, David and Jonas who handled everything from venue logistics to sponsorships
+- Our partners at **Bosch** and their invaluable contribution.
+- **Karlsruhe Mobility KAMO** for sponsoring the CAuDri banner
```
Comment (Copposhop) [inline _posts/2023-12-05-caudri-challenge-2023.md]: **Karlsruhe Mobility - KAMO** for their financial support

File: blog/archive.html [hunk 2]
```diff
@@ -1,10 +1,8 @@
 ---
 layout: default
 title: "Blog Archive"
-teaser: "Check out all blog posts in my blog archive. Click on a headline to read the teaser."
+teaser: "Check out all posts in our archive. Click on a headline to read the teaser."
 breadcrumb: true
```
Comment (Copposhop) [inline blog/archive.html]: Disable breadcrumb navigation bar?

**Review by Copposhop — COMMENTED** [review]
Shall we disable the breadcrumb navigation on the blog archive? 
And one small correction regarding the support from KAMO

**Review by Jajaho — COMMENTED** [review]
Disabled breadvrumb navigation and changed KAMO acknowledgement

**Review by Copposhop — APPROVED** [review]

**Pull request merged**